webroot/js/components/player.js

```diff
@@ -83,8 +83,7 @@ class OwncastPlayer {
     this.log('Start playing');
     const source = { ...VIDEO_SRC };
 
-    if (getLocalStorage(PLAYER_VOLUME) !== null)
-      this.vjsPlayer.volume(getLocalStorage(PLAYER_VOLUME));
+    this.vjsPlayer.volume(getLocalStorage(PLAYER_VOLUME) || 1);
```

Comment:
I think if the localStorage value = 0/mute, this will also end up being 1.

Reply:
I tested the functionality and it's working as expected.

![Peek 2020-10-07 00-09](https://user-images.githubusercontent.com/1775881/95289900-8d2eb400-0831-11eb-82bd-0884c3397651.gif)

Reply:
TIL localStorage keeps everything as strings, such as "0", which is truthy, so that's why this works. I guess this will do.  Thanks @SoldierCorp!